test/sql/updates/post.policies.sql

```diff
@@ -0,0 +1,6 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+SELECT * FROM _timescaledb_internal.bgw_job ORDER BY id;
```

Comment:
Not sure why this select is here. Could you elaborate?

Reply:
the update test diffs the output of the queries when run on fresh install against upgraded installation, this will catch any oversights in the job migration code

Reply:
Would be good to have a comment about that in the test.